data/ecosystems/m/mina-protocol.toml

```diff
@@ -434,6 +434,12 @@ url = "https://github.com/kriss1897/mina-google-oracle"
 [[repo]]
 url = "https://github.com/kriss1897/zk-social-recovery"
 
+[[repo]]
+url = "https://github.com/kaupangdx/dex-runtime"
+
+[[repo]]
+url = "https://github.com/kaupangdx/dex-ui"
+
```

Comment:
since you already added the repos in the child ecosystems, there's no need to add them to the parent. Can you please remove the repos from Mina?

Reply:
Done! :D